### PR TITLE
update variable names for consistency using standard formula terms;

### PR DESCRIPTION
**Pull request by lance-pyles**
fix flake8 syntax errors;

fix doctests;

### **Describe your change:**

* [ ] Add an algorithm?
* [x] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### **Checklist:**
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [x] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [x] All new Python files are placed inside an existing directory.
* [x] All filenames are in all lowercase characters with no spaces or dashes.
* [x] All functions and variable names follow Python naming conventions.
* [x] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [x] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [x] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by lance-pyles** [comment]
I used side because that is what wikipedia uses in the formula, A = s^2.  Reading the article closer, s is for side length, not just side.  Change is in, thanks.

File: maths/area.py [hunk 1]
```diff
@@ -48,24 +48,24 @@ def area_square(side_length):
     >>> area_square(10)
     100
     """
-    return side_length * side_length
+    return pow(side_length, 2)
```
Comment (cclauss) [inline maths/area.py:51]: This adds function call overhead but let's let it slide.

File: maths/area.py [hunk 2]
```diff
@@ -48,24 +48,24 @@ def area_square(side_length):
...
-    return 1 / 2 * length * breadth
+    return (base * height) / 2
 
 
 def area_parallelogram(base, height):
     """
     Calculate the area of a parallelogram
 
-    >> area_parallelogram(10,20)
+    >>> area_parallelogram(10,20)
```
Comment (cclauss) [inline maths/area.py:68]: 👍

File: maths/area.py [hunk 3]
```diff
@@ -75,8 +75,8 @@ def area_trapezium(base1, base2, height):
     """
     Calculate the area of a trapezium
 
-    >> area_trapezium(10,20,30)
-    450
+    >>> area_trapezium(10,20,30)
```
Comment (cclauss) [inline maths/area.py:78]: 👍

**Pull request merged**